### ISSUE TITLE
Wow, glad you're working on this!

### ISSUE THREAD

**Issue by jfrux** [Rastopyr/ember-cli-smart-tv#1]
Have an ember app that I'm wanting to put onto Tizen.
Any examples of doing this with your plugin?

**Comment by Rastopyr** [comment]
@joshuairl wow! we have ugly dummy application, and big technical debt 😄

**Comment by jfrux** [comment]
Okay... let me rephrase... I WISH you were working on this. :D
Do you know of a nice build setup for all TV platforms?

I've been using Samsung Toast and Cordova / Ember-cordova for a bit... but it just feels so clunky overall.  Would be nice to be able to package for Yahoo Connected TV, and even other platforms too.

**Comment by Rastopyr** [comment]
Suddenly not, i don't know :(

This package support only Samsung SmartTv with great nuances

**Comment by jfrux** [comment]
@Rastopyr Found any good solutions in the world of web for supporting large number of Smart TV's?
